### ISSUE TITLE
[suggestion] drag and drop outside the window

### ISSUE THREAD

**Issue by Vectasus** [TagStudioDev/TagStudio#75]
It would be nice if one could find a file in TagStudio and then drag it towards a new destination, such as a file explorer, a website drop area or a chatroom.

**Comment by Vectasus** [comment]
You can use `event.mimeData().hasUrls()` to detect files being dragged.
Here is a mock-up of a drop-target window
```python
import sys
from PySide6.QtWidgets import QApplication, QWidget
from PySide6.QtCore import Qt

class DragDropWindow(QWidget):
    def __init__(self):
        super().__init__()
        self.setAcceptDrops(True)
        self.setWindowTitle("Drag and Drop Files")
        self.setGeometry(100, 100, 400, 300)

    def dragEnterEvent(self, event):
        if event.mimeData().hasUrls():
            event.accept()
        else:
            event.ignore()

    def dragMoveEvent(self, event):
        if event.mimeData().hasUrls():
            event.accept()
        else:
            event.ignore()

    def dropEvent(self, event):
        for url in event.mimeData().urls():
            file_path = url.toLocalFile()
            print("Dropped file:", file_path)

if __name__ == "__main__":
    app = QApplication(sys.argv)
    window = DragDropWindow()
    window.show()
    sys.exit(app.exec_())
```

**Comment by CyanVoxel** [comment]
Implemented in #153
